contents/docs/libraries/next-js.md

```diff
@@ -323,7 +326,7 @@ export async function getServerSideProps(ctx) {
 
 ### App router
 
-For the app router, we can initialize the `posthog-node` SDK in every component we need it, or we can create a `PostHogClient` component to import into files like this:
+For the app router, we can initialize the `posthog-node` SDK once with a `PostHogClient` component to import into files like this:
```

Comment:
It's not technically a component though? A component is a UI Element? Im not super clued up on app routers though

```suggestion
For the app router, we can initialize the `posthog-node` SDK once with a `PostHogClient` component to import into files like this:
```